lib/contact.ts

```diff
@@ -203,7 +205,10 @@ interface CreateUserData extends CreateUserDataBase {
     externalId?: string;
 }
 
-type CreateLeadData = CreateUserDataBase;
+interface CreateLeadData extends CreateUserDataBase {
+    email?: string;
```

Comment:
If `email?` is pulled into the parent interface `CreateUserDataBase`, can it be removed from here and line 204 in `CreateUserData`?